app/views/users/index.html.erb

```diff
@@ -1,15 +1,19 @@
-<h1>Users</h1>
+<section>
 
-<p>Here is the list of users:</p>
+<div class="users-header">
+  <h2>Users list</h2>
+</div>
 
-<ul>
+<ul class="users-ul">
   <% @users.each do |user| %>
-  <li>
-    <div>User Image</div>
-    <div>
+  <li class="users-li">
+    <div><%= image_tag user.photo %></div>
+    <div class="user-info">
       <h2><a><%= user.name %></a></h2>
-      <span>Number of posts:<%= user.posts_counter %></span>
+      <p>Number of posts:<%= user.posts_counter %></p>
     </div>
   </li>
 <% end %>
 </ul>
```

Comment:
- [ ] I think it would be better if each view page design stick as much as possible to its wireframe design as required for **[the project](https://github.com/mullenlucas/blog-app-ror/pull/6/files#diff-dbe3a3b30b290647eb1f4c401b675c6c0e2f130fac1cadd25c1c447417a6b94bR12-R16)**